src/Client/AbstractStorePaymentMethodClient.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Client;
+
+abstract class AbstractStorePaymentMethodClient extends AbstractClient
+{
+    abstract public function getPaymentMethods(string $storeId);
```

Comment:
Return types seam to be missing here...

Reply:
For all 4 methods.

Reply:
The problem is the PaymentMethod is either LN or Onchain object. Typehinting both is only supported from PHP 8.0 afaik. Maybe better go with an Interface then?

Reply:
I can't really tell if an interface makes sense. If we have a lot of common fields, then sure, by all means.
But if the 2 objects are not similar, the interface would be quite empty. I haven't looked at these calls before, so I think you're the better judge here.
Thanks for clearing this up. I thought is was an oversight.